_includes/releases/v22.2/v22.2.7.md

```diff
@@ -0,0 +1,147 @@
+- Added support for the syntax [`CREATE DATABASE IF NOT EXISTS ... WITH OWNER`](../v22.2/create-database.html). [#97974][#97974]
+- Introduced a new internal virtual table `crdb_internal.node_memory_monitors`, which exposes all of the current reservations with the memory accounting system on a single node. Access to the table requires `VIEWACTIVITY` or `VIEWACTIVITYREDACTED` [permissions](../v22.2/security-reference/authorization.html#supported-privileges). [#97865][#97865]
+- Added a new [session setting](../v22.2/set-vars.html) `optimizer_always_use_histograms`, which ensures that the [optimizer](../v22.2/cost-based-optimizer.html) always uses histograms when available to calculate the [statistics](../v22.2/cost-based-optimizer.html#table-statistics) of every plan that it explores. Enabling this setting can prevent the optimizer from choosing a suboptimal index when statistics for a table are stale. [#98230][#98230]
+- Added a new aggregate [builtin function](../v22.2/functions-and-operators.html#array-functions) `array_cat_agg`. It behaves similarly to `array_agg(unnest(array_column))`: it takes arrays as its input and unnests them into array elements, which are then aggregated into a single result array. This is similar to concatenating all input arrays into a single one. [#98171][#98171]
+- Added the [cluster setting](../v22.2/cluster-settings.html) `sql.auth.modify_cluster_setting_applies_to_all.enabled` to regulate whether `MODIFYCLUSTERSETTING` can edit non `sql.defaults` settings. [#98234][#98234]
+- Fixed a bug where CockroachDB panicked when a user tried to [truncate](../v22.2/truncate.html) a table which has an ongoing [row-level TTL](../v22.2/row-level-ttl.html) change. CockroachDB still does not support table truncates in this scenario, but instead of panicking, an unimplemented error is returned. [#98591][#98591]
+- Added two views to the [`crdb_internal`](../v22.2/crdb-internal.html) catalog: `crdb_internal.statement_statistics_persisted`, which surfaces data in the persisted system, and `statement_statistics table` and `crdb_internal.transaction_statistics_persisted`, which surfaces the `system.transaction_statistics` table. [#98684][#98684]
+- The SQL Activity page now displays only persisted stats when selecting to [view fingerprints](../v22.2/ui-statements-page.html#statement-fingerprints-view). This means data recently executed might take up to 10 minutes to show on the [DB Console](../v22.2/ui-overview.html). [#99044][#99044]
+
+<h3 id="v22-2-7-operational-changes">Operational changes</h3>
+
+- Added a new [metric](../v22.2/metrics.html) `changefeed.schema_registry.retry_count`. This measures the number of request retries performed when sending requests to the [changefeed schema registry](../v22.2/changefeed-examples.html). Observing a non-zero value may indicate improper configuration of the schema registry or changefeed parameters. [#98349][#98349]
```

Comment:
Could you switch out the (../v22.2/metrics.html) link for monitor-and-debug-changefeeds.html. 

And switch out [changefeed schema registry](../v22.2/changefeed-examples.html), for:

changefeed [schema registry](../v22.2/stream-a-changefeed-to-a-confluent-cloud-kafka-cluster.html)